src/classes/CON_ContactMergeTDTM_TEST.cls

```diff
@@ -36,7 +36,8 @@
 
 @isTest
```

Comment:
This is a big test class and I think it just doubled. Do you think it's worth testing the `IsParallel=true` annotation to see if the tests in this class will support parallel testing? Alternatively, could we split the tests out so that one of the test classes is capable of running in parallel while the other isn't?

Reply:
None of the new tests that were added committed any data to the database so it shouldn't cause the test to run long. It also shouldn't cause the debug logs to get too big, but I can take a look at moving things around.

Reply:
I changed the tests in this class to use test setup data so that we can run in parallel. It is quite a bit more changes and there is one test that I had to comment out parts because when I made the changes there were unexpected results. I will comment in the code the line that is giving me grief.